2_base_and_sensor.ipynb

```diff
@@ -377,11 +685,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's use the sensor information in the program. Here, we will use the information from the laser scanner.\n",
+    "\n",
+    "**For advanced users** We use the following Python \"class\" to capture the sensor information. If you are interested, please take a look:\n",
```

Comment:
"**For advanced users**: We use the following Python \"class\" to capture the sensor information. If you are interested, please take a look:\n",